### PR TITLE
Adjust versions, bounds and CHANGELOGs to account for the latest release

### PR DESCRIPTION
**Pull request by lehins**
# Description

Recent release requires some adjustments on master.

# Checklist

- [x] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [x] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
      [versioning process](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#versioning-process).
- [x] The version bounds in `.cabal` files for all affected packages are updated. **If you change the bounds in a cabal file, that package itself must have a version increase.** (See [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#versioning-process))
- [x] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages. **New section is never added with the code changes.** (See [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd))
- [x] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
- [x] Cabal files are formatted (use `scripts/cabal-format.sh`)
- [x] [`hie.yaml`](https://github.com/input-output-hk/cardano-ledger/blob/master/hie.yaml) has been updated (use `scripts/gen-hie.sh`)
- [x] Self-reviewed the diff

### CODE REVIEW

File: eras/babbage/impl/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Version history for `cardano-ledger-babbage`
 
+## 1.4.3.1x
```
Comment (aniketd) [inline eras/babbage/impl/CHANGELOG.md]: Just noticed a stray `x` at the end of the version numbering here.

**Comment by aniketd** [comment]
If I understand correctly, we bump version numbers on `master` after every release, so that ongoing development can easily add changelog under the next bump, or bump it further, whichever is necessary?

**Comment by lehins** [comment]
> If I understand correctly, we bump version numbers on master after every release, so that ongoing development can easily add changelog under the next bump, or bump it further, whichever is necessary?

@aniketd Yes, this is the step 4 in the [release process](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#release-to-chap)

File: eras/babbage/impl/CHANGELOG.md (hunk 1 above)
Comment (lehins) [reply eras/babbage/impl/CHANGELOG.md]: Thank you. I got rid of it.

File: eras/conway/impl/CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,14 @@
 # Version history for `cardano-ledger-conway`
 
-## 1.7.0.0
+## 1.8.0.0
 
 * Make `DELEG`, `POOL` and `GOVCERT` conform to spec-v0.8 #3628
```
Comment (teodanciu) [inline eras/conway/impl/CHANGELOG.md:9]: These changes were merged before the release, so wouldn't they be in 1.7.0.0 ?

Comment (lehins) [reply eras/conway/impl/CHANGELOG.md:9]: They were merged before the release happened, but that doesn't mean they were included in the release :wink: 

In other words I prepared most of the release before that PR was merged, so it didn't make it in 1.7.0.0. That's why I needed to adjust the changelog

Comment (teodanciu) [reply eras/conway/impl/CHANGELOG.md:9]: But the [release commit](https://github.com/input-output-hk/cardano-haskell-packages/commit/3ffcc5a9f9520d9b0d6b95fbab485f62fc011367#diff-6c78dd71a5f166dde88662222ac941c14b1797d2cc69f8b4bb18fea09a5135b5) is `bbdfc3fdb6487653f23385c3df1c9c575bf567cf`, which is exactly the one introducing these changes, no? 

https://github.com/input-output-hk/cardano-ledger/commits/bbdfc3fdb6487653f23385c3df1c9c575bf567cf

**Review by aniketd — APPROVED** [review]
LGTM! :raised_hands:

File: eras/conway/impl/CHANGELOG.md (hunk 2 above)
Comment (lehins) [reply eras/conway/impl/CHANGELOG.md:9]: You are totally right. For some reason I was sure that I didn't include the latest PR into the release.

Fixed.

**Review by teodanciu — APPROVED** [review]
Looks good to me

**Pull request merged**